### PR TITLE
Jbh newiecunits

### PR DESCRIPTION
**Pull request by jhodgesatmb**

### CODE REVIEW

File: vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl [hunk 1]
```diff
@@ -1592,6 +1592,22 @@ qkdv:A0E0L2I0M0H-1T-2D0
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector> ;
   rdfs:label "A0E0L2I0M0H-1T-2D0" ;
 .
+qkdv:A0E0L2I0M0H-1T-3D0
+  a qudt:QuantityKindDimensionVector_ISO ;
+  a qudt:QuantityKindDimensionVector_Imperial ;
+  a qudt:QuantityKindDimensionVector_SI ;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
```
Comment (ralphtq) [inline vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl:1599]: qudt:dimensionExponentForAmountOfSubstance "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;

should be:

  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:integer ;

File: vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl [hunk 2]
```diff
@@ -1592,6 +1592,22 @@ qkdv:A0E0L2I0M0H-1T-2D0
...
+  qudt:dimensionExponentForElectricCurrent 0 ;
+  qudt:dimensionExponentForLength 2 ;
+  qudt:dimensionExponentForLuminousIntensity 0 ;
+  qudt:dimensionExponentForMass "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
```
Comment (ralphtq) [inline vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl:1603]: qudt:dimensionExponentForMass "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;

should be:

  qudt:dimensionExponentForMass  0 ;

File: vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl [hunk 3]
```diff
@@ -1592,6 +1592,22 @@ qkdv:A0E0L2I0M0H-1T-2D0
...
+  qudt:dimensionExponentForThermodynamicTemperature -1 ;
+  qudt:dimensionExponentForTime "-3"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
```
Comment (ralphtq) [inline vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl:1605]: qudt:dimensionExponentForTime "-3"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;

should be:

  qudt:dimensionExponentForTime -3 ;

**Comment by jhodgesatmb** [comment]
I typed into the form and didn’t check that it was a string. Thanks for catching this. Perhaps the form entry needs tweaking but I will fix the DVs.

Jack

File: vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl (hunk 1 above)
Comment (jhodgesatmb) [reply vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl]: fixed and pushed

File: vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl (hunk 2 above)
Comment (jhodgesatmb) [reply vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl]: fixed and pushed

**Pull request merged**